snap/snapcraft.yaml

```diff
@@ -29,6 +29,12 @@ apps:
     plugs:
       - network-bind
       - opengl
+    daemon: simple
+    install-mode: disable
+    restart-condition: on-failure
+    restart-delay: 2s
+    environment:
+      DCGM_HOME_DIR: "${SNAP_COMMON}"
```

Comment:
`DCGM_HOME_DIR` is an environment variable for `nv-hostengine`. I don't think we need it here. Though we do need to add env to allow custom metrics, we can do it in a separate task(perhaps with custom metrics)."
For this PR I think it is okay to simply remove it.

Reply:
Removed, thanks for clarifying